### PR TITLE
VEN-337 | Customer's application card

### PR DESCRIPTION
**Pull request by Nizar-Rahme**

### CODE REVIEW

File: src/locales/fi.json [hunk 1]
```diff
@@ -98,6 +98,26 @@
...
+      "boatType": "Veneen tyyppi",
+      "registrationNumber": "Rekisterinumero",
+      "boatWidth": "Veneen leveys",
+      "boatLength": "Veneen pituus",
+      "boatDepth": "Veneen syväys",
+      "boatWeight": "Veneen paino",
+      "boatName": "Nimi",
+      "boatBrand": "Merkki",
+      "selectedPorts": "VALITUT SATAMAT",
+      "choice": "Toive {{num}}",
```
Comment (Zeukkari) [inline src/locales/fi.json]: Can we move this template string to the application component?

File: src/common/checkbox/Checkbox.tsx [hunk 2]
```diff
@@ -4,23 +4,37 @@ import classNames from 'classnames';
 import styles from './checkbox.module.scss';
 import Icon from '../icon/Icon';
```
Comment (Zeukkari) [inline src/common/checkbox/Checkbox.tsx]: Can we switch to Icon component from HDS?

Comment (Nizar-Rahme) [reply src/common/checkbox/Checkbox.tsx]: The [Icon component](https://city-of-helsinki.github.io/helsinki-design-system/storybook/?path=/story/icons--iconcheck) from HDS won't work for this checkbox, but I can replace the Icon here with an imported SVG, so the checkbox would be a self-contained component as we discussed.
What do you think?

File: src/domain/individualCustomer/billsCard/BillsCard.test.jsx [hunk 3]
```diff
@@ -25,7 +25,7 @@ describe('BillsCard', () => {
   });
   const getWrapper = (props = mockProps) => shallow(<BillsCard {...props} />);
 
-  it('renders normally', () => {
+  it('renders correctly', () => {
```
Comment (Zeukkari) [inline src/domain/individualCustomer/billsCard/BillsCard.test.jsx]: Can we revert this?

**Pull request merged**